externs/es3.js

```diff
@@ -935,6 +935,14 @@ Boolean.prototype.toSource = function() {};
 Boolean.prototype.toString = function() {};
 
 /**
+ * @return {boolean}
+ * @nosideeffects
+ * @see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Boolean/valueOf
+ * @override
```

Comment:
Not sure if this `@override` will cause a problem. Looks like `Boolean` does not extend from anything.